gradle.properties

```diff
@@ -5,7 +5,7 @@ cordaProductVersion = 5.3.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 8
+cordaApiRevision = 9
```

Comment:
We only need to update this if we have a breaking change. I'd expect this change to be compatible with the existing runtime-os code.